website/integrations/services/grafana/index.mdx

```diff
@@ -78,6 +145,29 @@ token_url = https://authentik.company/application/o/token/
 api_url = https://authentik.company/application/o/userinfo/
 # Optionally map user groups to Grafana roles
 role_attribute_path = contains(groups[*], 'Grafana Admins') && 'Admin' || contains(groups[*], 'Grafana Editors') && 'Editor' || 'Viewer'
+```
+
+  </TabItem>
+  <TabItem value="helm">
+If you are using helm values-file instead, you have to set these options:
+
+```yaml
+grafana.ini:
+  auth:
+    signout_redirect_url: "https://authentik.company/application/o/<Slug of the application from above>/end-session/"
+    oauth_auto_login: true
+  auth.generic_oauth:
+    name: authentik
+    enabled: true
+    client_id: "<Client ID from above>"
+    client_secret: "<Client Secret from above>"
```

Comment:
For Helm values, I don't like telling users to put their client_secret in the values straight away. We should at least add a comment that says that this can also be passed as an environment variable with:
```yaml
envValueFrom:
  GF_AUTH_GENERIC_OAUTH_CLIENT_ID:
    secretKeyRef:
      name: grafana-oauth-credentials
      key: client_id
  GF_AUTH_GENERIC_OAUTH_CLIENT_SECRET:
    secretKeyRef:
      name: grafana-oauth-credentials
      key: client_secret
```

Reply:
Yes, I agree and the reasoning is sound !

however the reason I put it in there in the first place, and hardcoded values to match (instead of generated) was to make it easy to read and follow. I however have no strong feelings about it either way.

Let me see if I can make the changes, I am afraid I am not accustomed to the 'github part' of 'git' yet

Reply:
This gets a bit hairy ... I think there are 3 ways of doing this ... and if we pass it as env to Grafana it might break 
i will link the issue with discussion if I can find it again !

here it is [17363](https://github.com/helm/charts/issues/17363) could be old and outdated ...

So i am guessing thats the reason for the Grafana Helm docs to spec that you should add as a file ?

[Grafana Helm docs ](https://github.com/grafana/helm-charts/blob/main/charts/grafana/README.md#how-to-securely-reference-secrets-in-grafanaini)

I will look into it and try it before we doc it ... in my case I dodge the issue using 

#   $ openssl rand -hex 32

which would mean there are spec chars to break the env import, do we have any such guarantees if go with generated values ? Would that be a better option ?

Reply:
We're using that way for our grafana instance so I'm pretty sure it works. We can also not specify a specific way of doing things, but just have a comment along the lines of "The client secret should not be passed directly via Helm values, but via a secret. Check Grafana Helm chart documentation for more information."